ui/lib/apps/TopSQL/utils/useTimeWindowSize.ts

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react'
+
+export const createUseTimeWindowSize = (barWidth: number) => {
```

Comment:
Could you add comments about what is this used for?

Reply:
Yes!